packages/@cra-express/universal-loader/src/universal.js

```diff
@@ -6,8 +6,18 @@ const stringRenderer = require('./renderer/string-renderer').default;
 const craServiceName = process.env.CRA_SERVICE_NAME || 'localhost';
 const craClientPort = process.env.CRA_CLIENT_PORT || 3000;
 
+function resolveHtmlFilenameByRequest(req, options) {
+  var defaultHtml = 'index.html';
```

Comment:
I prefer this to be outside the function and as a `const`

Reply:
otherwise, it's all good 👍

Reply:
you're right, it's better that way